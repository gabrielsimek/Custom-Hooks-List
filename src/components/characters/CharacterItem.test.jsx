import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterItem from './CharacterItem';
import { MemoryRouter } from 'react-router-dom';
describe('App CharacterItem', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<MemoryRouter>
      <CharacterItem /></MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});
