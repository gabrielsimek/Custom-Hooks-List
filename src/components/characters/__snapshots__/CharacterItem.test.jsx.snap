// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App CharacterItem renders App 1`] = `
<DocumentFragment>
  <figure>
    <img />
    <figcaption>
      <h2 />
      <p />
      <p />
    </figcaption>
  </figure>
</DocumentFragment>
`;
